wagtail/wagtailusers/views/users.py

```diff
@@ -27,40 +27,42 @@
 def index(request):
     q = None
     is_searching = False
+    model_fields = User._meta.get_all_field_names()
```

Comment:
This method was deprecated in Django 1.8: https://github.com/django/django/blob/ae1d663b7913f6da233c55409c4973248372d302/django/db/models/options.py#L560-L561